### DISCUSSION TITLE
Deferred Loading (feature request)

### DISCUSSION THREAD

**Discussion by ActionCoding** [fiduswriter/Simple-DataTables#320] [Ideas]
I would like to see a deferred loading feature added if possible. Similar to [here](https://datatables.net/examples/server_side/defer_loading.html)

I do not see anything in the docs on how this would be done. Am i missing something or is this not an implemented feature currently?

**Comment by johanneswilm** [comment]
Datatables.net builds on jquery, which is from another time where JavaScript was written very differently.

simple-datatables doesn't make any server calls for you. You have to make these yourself. Due to the difference the way that rendering will be deferred, etc. will likely also be quite different and there will be many different situations that all differ slightly so a general approach to this can probably not be programmed the way it could back in the jQuery/datatables.net days.

If you have that exact same situation where you first want to load X entries and then later on Y more, you should be able to initially just parse it Y empty entries and then later switch out `datatables.data.data`.
